SETI/rms-opus
# 1153 update import to handle new pdsfile changes

- Fixes #1153 
- Were any Django, import pipeline, table_schema, or dictionary files modified? Y
  - If YES:
    - Database used: re-import on a test database using import_for_tests.sh
    - All Django tests pass: Y
    - FLAKE8 run on modified code: Y
- Were any JavaScript or CSS files modified? N
  - If YES:
    - JSHINT run on all affected files: NA
    - Tested on Chrome, Firefox, Safari, Edge, and iOS: NA
      (Remember to test all browser sizes)
    - Tested for race conditions (with delayed API calls if applicable): NA

Description of changes:
- Update tests results for STIS (wavelength values is changed after re-import)
- Make sure all the opus_type & opus_prioritizer in pdsfile rules (latest master branch in pds-webtools) are in internal_def_product_types.json. The newly added opus_type is "gossi_raw_alternate". Tooltips in detail tab for GOSSI superseded processing files shows up after re-import.
- For NH volset, update to make sure 2001 is imported first and then 1001 after that. Update the tests results for NH after re-import.

Known problems:
NA
